cypress/e2e/client-tasks/pagination.cy.js

```diff
@@ -11,17 +11,27 @@ describe("Pagination", () => {
           .invoke('val').should('contain', 'per-page=25')
     })
 
-    it("will not show previous arrow on page 1 but will show it on other pages", () => {
+    it("will not show previous on page 1 but will show it on other pages", () => {
       cy.get(".previous-page-pagination-link").should('not.exist')
-      cy.get("#top-nav .next-page-pagination-link").click()
+      cy.get(`[aria-label="Page 2"]`).first().click()
+      cy.get(".previous-page-pagination-link").should('be.visible', 'Previous')
+      cy.get(`[aria-label="Page 3"]`).first().click()
+      cy.get(".previous-page-pagination-link").should('be.visible', 'Previous')
+      cy.get(`[aria-label="Page 4"]`).first().click()
+      cy.get(".previous-page-pagination-link").should('be.visible', 'Previous')
+      cy.get(`[aria-label="Page 5"]`).first().click()
```

Comment:
what's the value in checking pages 3, 4 and 5?

Reply:
I guess checking that for some reason the previous button doesn't disappear? Happy to take it out if you think its overkill